cmd/outline-ss-server/metrics.go

```diff
@@ -49,9 +62,106 @@ type outlineMetrics struct {
 var _ service.TCPMetrics = (*outlineMetrics)(nil)
 var _ service.UDPMetrics = (*outlineMetrics)(nil)
 
+type ReportTunnelTimeFunc func(IPKey, ipinfo.IPInfo, time.Duration)
+
+type activeClient struct {
+	mu              sync.Mutex
+	IPKey           IPKey
```

Comment:
I think we can remove the key from the struct, since it's already in the key. No point in having it here too.